app/models/event.rb

```diff
@@ -31,4 +33,13 @@ def attended_users
   def absent_users
     self.attendances.where(present: false).collect(&:user)
   end
+
+  # Search #####################################################################
+
+  def self.find_finable_event_ids
+    Event.where("date < :today", today: DateTime.today)
+         .where.not(fine: nil)
+         .pluck(:id)
```

Comment:
Style/DotPosition: Place the . on the previous line, together with the method call receiver.